experimental/packages/opentelemetry-api-metrics/src/types/Meter.ts

```diff
@@ -80,6 +80,9 @@ export interface Meter {
 
   /**
    * Creates a new `ObservableGauge` metric.
+   *
+   * The callback SHOULD not be called concurrently.
```

Comment:
This sounds like something we are telling the user not to do when it is the SDK that actually calls the callback

Reply:
Yeah, maybe we can update the words to be like "The callback is been called only when it finishes the previous invocation or timeouts."